generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs

```diff
@@ -53,7 +53,7 @@ const LAYOUT_ROUTES = [navbarRoute, ...errorRoute];
         <%_ } _%>
         ...LAYOUT_ROUTES,
       ],
-      { enableTracing: DEBUG_INFO_ENABLED, relativeLinkResolution: 'corrected' }
+      { enableTracing: DEBUG_INFO_ENABLED }
```

Comment:
squash this commit with the first one of this PR, as relativeLinkResolution has been added and then removed ?

Reply:
done